specification/schemas/return-methods/ReturnMethod.json

```diff
@@ -20,9 +20,17 @@
             "default_weight": {
               "$ref": "#/components/schemas/Weight"
             },
-            "name": {
+            "title": {
```

Comment:
Shall we remove the `"additionalProperties": false` above to make old requests not fail on the spec?

Reply:
Good call, I thought about this after I made the PR but forgot to bring up the question. I wondered though, since we're the only consumers of this API, if we could get away with this (breaking) change and keep the `additionalProperties` property?

Reply:
If all our code is updated within the same release, then that will be fine 👌🏻